console/console-count-logging.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Test console log are present</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script>
+    promise_test(async test => {
+        await test_driver.bidi.log.entry_added.subscribe();
+
+        const log_entries = [];
+        // Collect all the `count` log entries' texts into `log_entries`.
+        test_driver.bidi.log.entry_added.on(event => {
+                if (event.method === 'count') {
+                    log_entries.push(event.text);
+                }
+            }
+        );
+
+        console.count();
+        console.count(undefined);
+        console.count("default");
+        console.count({
+            toString() {
+                return "default"
+            }
+        });
+
+        await test.step_wait(() => log_entries.length >= 4,
```

Comment:
I think the test looks good but I think time-based polling somewhat defeats the purpose of events. Would it be possible to change step_wait to accept an asynchronious condition function instead? Alternatively, it could be beneficial to have another help that creates a promise with a timeout and integrates with test-harness internals.

Reply:
agree. I made a work around in the test, as adding it in testharness requires RFC and is out of scope of this PR